backend/compact-connect/lambdas/python/provider-data-v1/tests/function/test_handlers/test_providers.py

```
@@ -376,6 +380,23 @@ def test_get_provider_returns_expected_general_response_when_caller_does_not_hav
             scopes='openid email aslp/readGeneral', expected_provider=expected_provider
         )
 
+    def test_get_provider_returns_expected_download_links_when_caller_is_compact_admin(self):
+        self._load_provider_data()
+
+        provider_data = self._call_get_provider_and_return_provider_data(
+            scopes='openid email aslp/readGeneral aslp/admin aslp/readPrivate'
+        )
+
+        self.assertEqual(
+            'military-waiver.pdf', provider_data['militaryAffiliations'][0]['downloadLinks'][0]['fileName']
+        )
+        # we can't assert on the whole url, since it changes with time
+        # we can verify the path to the file matches expected values
+        self.assertIn(
+            'https://provider-user-bucket.s3.amazonaws.com//provider/89a6377e-c3a5-40e5-bca5-317ec854c570/document-type/military-affiliations/2024-07-08/1234%23military-waiver.pdf',
+            provider_data['militaryAffiliations'][0]['downloadLinks'][0]['url'],
+        )
+
```

Hmm, isn't this testing (basically) that the URLs match, contradicting your comment? (I would expect this to fail, but I must be missing something.)